### DISCUSSION TITLE
Is there any way where I can see all errors together of my project?

### DISCUSSION THREAD

**Discussion by ambujkhanna** [streetsidesoftware/vscode-spell-checker#1511] [General]
Hi,

I am curious to know about a feature/option where I can see or export list of error available in my project directory.

Is there any way or command which can check spell correction errors in entire directory and provide me report of it?

Thanks in advance!

**Comment by Jason3S** [comment]
@ambujkhanna,

The best way is to use the [cspell](https://www.npmjs.com/package/cspell)-cli. It can scan your entire project for errors.
